creator-node/scripts/sequelize.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
```

Comment:
I would like to keep this file and possibly merge it. I think it's actually very useful going forward because this allows us to init a sequelize instance with models attached against the dbUrl. If we attempted to import the other sequelize init, we get a bunch of errors because of the convict validation